Makefile

```diff
@@ -3,24 +3,25 @@ ROOT_DIR=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 # Build everything after cleaning
 all:	clean	ui	antora
 
+# Rule to clean cache
+clean:
+	rm -rf ui/build
+	rm -rf playbook/build
+
 # Rule to build the UI before building the docs
-ui:
+ui: clean
```

Comment:
Are you using tab for the Makefile @Vandit1604? We are not supposed to use a single space here, so would like to double check with you